spec/rest_in_peace/template_sanitizer_spec.rb

```diff
@@ -120,12 +120,20 @@
 
       context 'param which need to be encoded' do
         let(:params) { { id: 'this param need to be encoded' } }
-        let(:encoded_param) { 'this+param+need+to+be+encoded' }
+        let(:encoded_param) { 'this%20param%20need%20to%20be%20encoded' }
```

Comment:
Case 3 is also working as expected.

-  **3**: `/path/with a space/?query=with a space`
   - should become /path/with%20a%20space/?query=with+a+space
    - but it's currently /path/with+a+space/?query=with+a+space